Source/Peripheral.swift

```diff
@@ -165,7 +165,7 @@ public class Peripheral {
     public func discoverIncludedServices(_ includedServiceUUIDs: [CBUUID]?, for service: Service) -> Observable<[Service]> {
```

Comment:
Same things here... Maybe we can reuse above code in better way. Not just copy paste and change few things around.